### PR TITLE
Toothsome table: Adding Contributing.md file

### PR DESCRIPTION
**Pull request by ttseng**
Documenting the processing Allyson and I went through to try to update the NPM package.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Update the NPM Version 
+
+Open the [shared-components app](https://shared-components.glitch.me) on Glitch, then run
+
+```
+# 1. Update the NPM version
+npm version patch
+
+# 2. Update changelog.md with your changes
```
Comment (modernserf) [inline CONTRIBUTING.md]: might want to add a note here that you need to `refresh` the container to see the placeholder changes in changelog.md

Comment (ttseng) [reply CONTRIBUTING.md]: good idea - added!

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Pull request merged**